twikit/x_client_transaction/transaction.py

```
@@ -31,12 +31,21 @@ async def init(self, session, headers):
-        self.key = self.get_key(response=self.home_page_response)
-        self.key_bytes = self.get_key_bytes(key=self.key)
-        self.animation_key = self.get_animation_key(
-            key_bytes=self.key_bytes, response=self.home_page_response)
+        try:
+            self.DEFAULT_ROW_INDEX, self.DEFAULT_KEY_BYTES_INDICES = await self.get_indices(
+                self.home_page_response, session, headers)
+            self.key = self.get_key(response=self.home_page_response)
+            self.key_bytes = self.get_key_bytes(key=self.key)
+            self.animation_key = self.get_animation_key(
+                key_bytes=self.key_bytes, response=self.home_page_response)
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)